### PR TITLE
Added timestamp col, updated select query to order by timestamp

### LINKED ISSUE #44: Add datetime column to sqlite YStore
We are finding situations where it would be useful to order updates by date/time. One example is to clean out or combine updates of older documents. The idea is that a YStore subclass could order by room id and sort by date/time and pick the last entry to find the last time the document was edited. If that is longer than a certain amount of time (say a week) we could delete the history of that document or collapse it into a single update.

The proposal is to add a new date/time column to the database and to compute the date/time in the write method.

### PR DESCRIPTION
**Pull request by 3coins**
Fixes #44 

cc @ellisonbg @dlqqq

### CODE REVIEW

File: ypy_websocket/ystore.py [hunk 1]
```diff
@@ -162,5 +163,5 @@ async def write(self, data: bytes) -> None:
         await self.db_created.wait()
         metadata = await self.get_metadata()
         async with aiosqlite.connect(self.db_path) as db:
-            await db.execute("INSERT INTO yupdates VALUES (?, ?, ?)", (self.path, data, metadata))
+            await db.execute("INSERT INTO yupdates VALUES (?, ?, ?, ?)", (self.path, data, metadata, datetime.now()))
```
Comment (davidbrochart) [inline ypy_websocket/ystore.py:166]:
```suggestion
            await db.execute("INSERT INTO yupdates VALUES (?, ?, ?, ?)", (self.path, data, metadata, datetime.utcnow()))
```

**Comment by davidbrochart** [comment]
Thanks @3coins!
What do we plan to do for the file-based YStore?

File: ypy_websocket/ystore.py (hunk 1 above)
Comment (ellisonbg) [reply ypy_websocket/ystore.py:166]: This doesn't affect the public APIs, so I don't think we need to address it (at least for now). By that, I mean a subclass of the sqlite YStore could do something with this column, but it isn't part of the public API.

File: ypy_websocket/ystore.py [hunk 2]
```diff
@@ -147,7 +148,7 @@ async def read(self) -> AsyncIterator[Tuple[bytes, bytes]]:  # type: ignore
         try:
             async with aiosqlite.connect(self.db_path) as db:
                 async with db.execute(
-                    "SELECT yupdate, metadata FROM yupdates WHERE path = ?", (self.path,)
+                    "SELECT yupdate, metadata FROM yupdates WHERE path = ? order by datetime(timestamp) desc", (self.path,)
```
Comment (ellisonbg) [inline ypy_websocket/ystore.py:151]: I don't think we need to order by the timestamp in the read.

**Review by ellisonbg — CHANGES_REQUESTED** [review]

**Comment by 3coins** [comment]
Updated as per comments, also added the pre-commit updates.

**Comment by 3coins** [comment]
> Thanks @3coins!
> What do we plan to do for the file-based YStore?
@davidbrochart I haven't really thought about the file store. Does it have a defined schema that we can update to add a similar attribute for timestamp?

File: ypy_websocket/ystore.py (hunk 2 above)
Comment (3coins) [reply ypy_websocket/ystore.py]: I updated and removed this based on your comments. However, how do we plan to use the newly added timestamp, if not adding to the select statement here?

Comment (ellisonbg) [reply ypy_websocket/ystore.py]: For now, a subclass of this YStore would issue its own queries using the new column.

**Comment by ellisonbg** [comment]
I don't think we need to handle this in the file based YStores until the timestamp is part of the public API of YStore.

**Review by ellisonbg — APPROVED** [review]

**Comment by hbcarlos** [comment]
Thanks, @3coins!

**Pull request merged**